api/src/main/java/io/opencensus/trace/unsafe/ContextUtils.java

```diff
@@ -33,7 +33,7 @@
  *
  * @since 0.5
  */
-public final class ContextUtils {
```

Comment:
Hiding `ContextUtils` to force everyone to migrate to `CtxUtils` or use other abstractions like `CurrentSpanUtils`.